code/mobility-services/mobility-services.js

```diff
@@ -189,4 +198,44 @@ $(document).on("knack-view-render.any", function (event, page) {
 
     customizeLoginButton(viewId);
   }
-});
\ No newline at end of file
+});
+
+/****************************************/
+/*** Dropdown Menu Buttons Navigation ***/
+/****************************************/
+function dropdownMenuItem(recordId, route, linkName) {
+  return (
+    `<li class="kn-button">\
+      <a href="#application-operating-authority/business-information/${recordId}/${route}/${recordId}">\
+        <span>${linkName}</span>\
+      </a>\
+    </li>`)
+}
+
+// Dictonary of views needing dropdown menu in editable Operating Authority (OA) pages
+var viewNameOA = {687:"1 - Service Type",676:"2 - Business Information",689:"3 - Insurance", 
+691:"4 - Authorized Person", 693:"5 - Vehicle Information", 695:"6 - Review and Submit"};
+
+for (let key in viewNameOA) {
+  $(document).on('knack-view-render.view_' + key, function(event, view, record) {
```

Comment:
double check that your render function also works for mobile or you'll need a separate one. I like the reusability of the view keys for using your menu across multiple pages.